test/e2e/resilience/resilience_multizone_k8s.go

```diff
@@ -16,7 +16,11 @@ func ResilienceMultizoneK8s() {
 		// Global
 		global = NewK8sCluster(NewTestingT(), Kuma1, Silent)
 		Expect(NewClusterSetup().
-			Install(Kuma(core.Global, WithCtlOpts(map[string]string{"--set": "controlPlane.terminationGracePeriodSeconds=5"}))).
+			Install(Kuma(core.Global,
+				WithCtlOpts(map[string]string{"--set": "controlPlane.terminationGracePeriodSeconds=5"}),
+				WithEnv("KUMA_GENERAL_RESILIENT_COMPONENT_BASE_BACKOFF", "1s"),
+				WithEnv("KUMA_GENERAL_RESILIENT_COMPONENT_MAX_BACKOFF", "10s"),
```

Comment:
I'd be even more aggresive here. for example: 1s and 1s. It can speed up tests.

Reply:
changed